### DISCUSSION TITLE
Error on running NNConv

### DISCUSSION THREAD

**Discussion by shrimonmuke0202** [pyg-team/pytorch_geometric#5116] [Q&A]
Hi all,
When I apply NNConv on a graph whose shape of edge attribute is (# of edges, 50) gives me a runtime error at 
``weight = weight.view(-1, self.in_channels_l, self.out_channels)``. How can I solve this error? Also can you gave me an example how to use NNConv?

**Comment by wsad1** [comment]
Could you post the full error message. And also share what you set `nn` to in `NNConv`.  This will help us answer you better.

**Reply by rusty1s** [reply]
+1 In addition, [here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/mnist_nn_conv.py#L37-L43) is an example on how to use `NNConv`.

**Comment by shrimonmuke0202** [comment]
Thanks for your answer.
